src/commands/utils/color.ts

```diff
@@ -3,19 +3,30 @@ import sharp from "sharp";
 import { SlashCommand } from "../_types";
 import { ColorTranslator } from "colortranslator";
 
+const customColors: {
+	[key: string]: string;
+} = {
+	transparent: "#00000000",
```

Comment:
Just did `transparent` so far and I could make the if statement just check for that but this just puts it in place if we want to add another extra color in the future.